Gemfile

```diff
@@ -1,4 +1,4 @@
-source 'https://rubygems.org'
+source 'https://ruby.taobao.org'
```

Comment:
@plribeiro3000 I change it as I am in China and the GFW blocks rubygems.org :(
But I forgot to change it back.

I will change it back shortly!